src/plugins/shared_ux/public/components/empty_state/no_data_views/index.tsx

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { NoDataViewsPage } from './no_data_views_page';
+export { NoDataViewsPage } from './no_data_views_page';
+
+// eslint-disable-next-line import/no-default-export
+export default NoDataViewsPage;
```

Comment:
nit: I was recently convinced to change my approach to imports by other Kibana engineers, given the noise of disabling eslint.

This would be in [src/plugins/shared_ux/public/components/index.ts](https://github.com/elastic/kibana/pull/125403/files#diff-aa2be68c12725b21221ed34b699606f8d0a3a427c4a5b320c3e27d5a194a0ec7)

```
export const NoDataViewsPage = React.lazy(() =>
  import('./no_data_views_page').then(({ NoDataViewsPage }) => ({
    default: NoDataViewsPage,
  }))
);
```